nemo/collections/common/metrics/loss.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from pytorch_lightning.metrics import Metric
+
+__all__ = ['Loss']
+
+
+class Loss(Metric):
```

Comment:
The name is too generic. It's supposed to be a metric not an actual loss function / class. Please call it something like GlobalAverageLossMetric or something similar to denote that 1) it is a metric, not an actual loss function 2) that it computed true global average and not running average of a metric. 

Same with the filename